views/home.njk

```diff
@@ -7,34 +7,36 @@
       <div class="usa-grid">
         <div class="usa-hero-callout usa-section-dark">
           <h1>
-            Federalist compliantly hosts federal government websites.
+            Launch a site in 30 minutes. 
           </h1>
-          <p><a class="usa-hero-link" href="https://docs.google.com/forms/d/1iB8aW7c9r1QH3s8XElQCrnXRGjAiPUYpWG1CMeEqGIo/viewform">Sign up for more information</a></p>
+          <p>Select one of our pre-built designs and launch a fully compliant website for your federal agency, program or office today.</p>
+          <p>
+          <p><a class="usa-button" href="https://docs.google.com/forms/d/1iB8aW7c9r1QH3s8XElQCrnXRGjAiPUYpWG1CMeEqGIo/viewform">Get started</a></p>
         </div>
       </div>
     </div>
     <div class="build-count">
       <div class="usa-grid text-center">
-        <p>{{ buildCount }} site updates deployed in the past week.</p>
+        <p>Number of site updates this week: {{ buildCount }}</p>
       </div>
     </div>
     <div class="usa-grid">
       <div class="usa-width-one-third home-feature">
         <div class="feature-text">
-          <h2><img src="/images/lock.svg"/> Secure and scalable</h2>
-          <p>Federalist websites offer the highest level of security and scalability. They are generated static sites hosted on FedRAMP certified cloud infrastructure.</p>
+          <h2><img src="/images/layers.svg"/> No experience necessary</h2>
```

Comment:
I don't know if we can say that for Chris the Communicator. Most customers do need HTML/front end experience do accomplish their needs, unless their needs are simple.